modules/local/lookup_kingdom.nf

```diff
@@ -0,0 +1,26 @@
+process LOOKUP_KINGDOM {
+
+    tag "${meta.prefix}"
+
+    label 'process_light'
+
+    container 'quay.io/microbiome-informatics/genomes-pipeline.python3base:v1.1'
```

Comment:
I'll create a different base container, not linked to the genomes-pipeline (but we can swap that later)